_posts/2017-10-15-ABRT-ansible-faf-url-setup.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "FAF url setup of ABRT client Ansible role"
+date:   2017-10-15 10:31:14 +0100
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt ansible faf url setup
+---
+We recently added a new option to ABRT client [Ansible role][ansible-link] to setup
+url of the FAF server, i.e. where crash reports from ABRT client are reported.
+
+This small improvement will mostly appreciate people using or willing to use their own
+installation of FAF for gathering crash reports from ABRT client for custom analysis.
+While their own FAF installation is running on custom server (easy to do using
+[FAF ansible role][faf-ansible-link]) or have their FAF running [in container][faf-in-container-link].
+
+# Usage
+To use ABRT client Ansible role, it is needed to simply declare it in your playbook, as:
```

Comment:
To use ABRT client Ansible role it is needed to declare it ...
(comma, and simply)